src/lustre/lustrePath.ml

```diff
@@ -1802,21 +2046,33 @@ let rec pp_print_lustre_path_json' is_top const_map ppf = function
           Format.fprintf ppf ",@,\"subnodes\" :@,[@[<v 1>%a@]@,]"
             (pp_print_lustre_path_json' false const_map) subnodes
     in
-
+      let rec interleave = function 
+      | (l, []) | ([], l) -> l 
+      | (x::xs, y::ys) -> x::y::(interleave (xs,ys))
+      in
```

Comment:
This function is duplicated; it is also defined globally. Remove this instance and use the global definition.